### ISSUE TITLE
could we get the training details for the label adaptation process? 

### ISSUE THREAD

**Issue by mshooter** [microsoft/FaceSynthetics#49]

**Comment by TadasBaltrusaitis** [comment]
Hi,

The process is as follows

We first train a model on synthetic data alone (e.g. landmarks or segmentation), this model never sees any real data and is the one we are evaluating.  We use the trained model on a real dataset (e.g. training and validation sets) for our target task to acquire synthetic convention predictions.

We now have matching pairs of items, "synthetic predictions in synthetic data convention" and "ground truth annotations in human annotator convention". We can now train a network to go from synthetic convention to human convention. We use a Unet for segmentation and an MLP for landmarks. We train using adam optimizer for 600 epochs and pick the best performing model on validation data.

We can now evaluate the models by comparing predictions from translator to ground truth `prediction = translator(sx_network(image))`

Hope this helps!
